core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -236,57 +227,58 @@ message GetMutationsResponse {
   string next_page_token = 7;
 }
 
-// GetDomainInfoRequest contains an empty request to query the GetDomainInfo
-// APIs.
-message GetDomainInfoRequest {
-  // domain_id identifies the domain in which the user and application live.
-  string domain_id = 1;
-}
-
-// GetDomainInfoResponse contains the results of GetDomainInfo APIs.
-message GetDomainInfoResponse {
-  // domain_id identifies the domain in which the user and application live.
-  string domain_id = 4;
-  // Log contains the Log-Tree's info.
-  trillian.Tree log = 1;
-  // Map contains the Map-Tree's info.
-  trillian.Tree map = 2;
-  // Vrf contains the VRF public key.
-  keyspb.PublicKey vrf = 3;
-}
-
-// GetEpochsRequest is an empty proto message used as input to GetEpochs API.
-message GetEpochsRequest {}
-
-// GetEpochsResponse contains mutations of a newly created epoch.
-message GetEpochsResponse {
-  // mutations contains all mutations information of a newly created epoch.
-  GetMutationsResponse mutations = 1;
-}
-
 // The KeyTransparencyService API represents a directory of public keys.
 // The API defines the following resource model:
-//
-// - The API has a collection of [Entry][keytransparency.Entry]
-//   resources named `/user/*`.
-//
-// - Entries have a collection of historical
-//   [Entry][keytransparency.Entry] resources named `/users/*/history`.
-//
+// - The API has a collection of domains named `/v1/domains/`.  
+//   Each domain has a single sparse merkle tree, append only log, 
+//   and other public key material that is needed to verify server responses .
+// - Each Domain has a collection of snapshots, called epochs named `/v1/domains/*/epochs/`.
+//   Each Epoch contains the root of the sparse merkle tree and the changes that 
+//   occured that epoch and the previous.  
+// - Each Domain has a collection of applications named `/v1/domains/*/apps/`.
+//   Each Application has distinct users and permissions, but are indistinguishable 
+//   from eachother just by looking at the tree.
+// - Each Application has a collection of Users named `/v1/domains/*/apps/*/users`.
+//   Each User contains public key material, permissions for who is allowed to change 
+//   that user, and signatures indicating who made the last change.  
+// - Each User also has a collection of historical values for user named   
+//   `/v1/domains/*/apps/*/users/*/history`.
```

Comment:
The resource model description is great.  It's inherently hierarchical, so the comments could reflect that:

// The API has a collection of domains:
// `/v1/domains/`
//      * each domain has a single sparse Merkle tree, append-only log,
//        and other public key material ...
//        Each domain has a collection of snapshots, called Epochs
// `/v1/domains/*/epochs/`
//      * each epoch contains the root of the sparse Merkle tree...
//         etc.
// `/v1/domains/*/apps/`
//         etc.
// `/v1/domains/*/apps/*/users`
//         etc.
// `/v1/domains/*/apps/*/users/*/history`
//         etc.

Reply:
Thanks for this suggestion.